apps/api/src/domain/models/digital-text/commands/create-digital-text.command.ts

```diff
@@ -0,0 +1,48 @@
+import { ICommandBase, LanguageCode } from '@coscrad/api-interfaces';
+import { Command } from '@coscrad/commands';
+import { NestedDataType, NonEmptyString, UUID } from '@coscrad/data-types';
+import { LanguageCodeEnum } from '../../../common/entities/multilingual-text';
+import { AggregateCompositeIdentifier } from '../../../types/AggregateCompositeIdentifier';
+import { AggregateType } from '../../../types/AggregateType';
+import { AggregateTypeProperty } from '../../shared/common-commands';
+
+export class DigitalTextCompositeId {
+    @AggregateTypeProperty([AggregateType.digitalText])
+    type = AggregateType.digitalText;
+
+    @UUID({
+        label: 'ID',
+        description: 'unique identifier',
+    })
+    id: string;
+}
+
+@Command({
+    type: 'CREATE_DIGITAL_TEXT',
+    label: 'Create Digital Text',
+    description: 'Creates a new digital text',
+})
+export class CreateDigitalText implements ICommandBase {
+    @NestedDataType(DigitalTextCompositeId, {
+        label: 'Composite Identifier',
+        description: 'system-wide unique identifier',
+    })
+    readonly aggregateCompositeIdentifier: AggregateCompositeIdentifier<
+        typeof AggregateType.digitalText
+    >;
+
+    @NonEmptyString({
+        isOptional: false,
+        label: 'title',
+        description: "digital text's title in the language",
```

Comment:
Reword this. `the language` does not refer to "whatever language code you selected". Rather, it refers to "the Indigenous language for your tenant". Maybe
"digital text's title in the given language" ? or
"digital text's title in selected language"?
I imagine the original language could commonly be English or Tŝilhqot’in \ Haida, or even occasionally French for historical records or if some day a group out east signs on.

Reply:
I updated this and also the corresponding description in `CreateSong`.